### PR TITLE
replacing "compliant" with "conformant"

### PR DESCRIPTION
**Pull request by jhdalek55**
Following a discussion at the 7/19 Uptane Standards meeting, it was decided we should not be using the term "Uptane-compliant." As a general rule of thumb "compliance" is reserved for regulations. So, I've removed the seven mentions of the phrase and replaced it with the term "Uptane-conformant."

We may also want to add text to discuss the issue further at some point.

### CODE REVIEW

**Comment by jhdalek55** [comment]
@tkfu @plapczyn  please review

**Review by trishankatdatadog — APPROVED** [review]

**Comment by jhdalek55** [comment]
Thanks @trishankatdatadog. Could we get one more approval and a merger? We want to add a more explicit definition of conformance to this introduction, but I would prefer to do that after we merge the revised terminology present in this PR.

**Review by mnm678 — APPROVED** [review]

**Comment by jhdalek55** [comment]
Thanks @mnm678 .

**Pull request merged**
